### PR TITLE
[15.0][MIG] stock_move_line_auto_fill: Migration to version 15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT36555

ping @CarlosRoca13 @carlosdauden 

this PR supersedes https://github.com/OCA/stock-logistics-workflow/pull/1111

### CODE REVIEW

File: stock_move_line_auto_fill/tests/test_stock_picking_auto_fill.py [hunk 1]
```diff
@@ -444,6 +444,8 @@ def test_return_twice(self):
         # Covered case:
         # Return more than one times not duplicate quant units in stock
         self.picking_type_out.auto_fill_operation = True
+        # By default in v15 there is an opertion type to return goods
```
Comment (rousseldenis) [inline stock_move_line_auto_fill/tests/test_stock_picking_auto_fill.py]: `# By default from v15 there is an operation type to return goods`

**Comment by sergio-teruel** [comment]
@rousseldenis Changes done.

I have added an extra commit to fix the tests due to a missing commit from v13.0

**Review by carlosdauden — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**